WcaOnRails/app/views/static_pages/delegates.html.erb

```diff
@@ -5,9 +5,7 @@
   <p><%= t('about.structure.delegates_html', see_link: "") %></p>
   <p><%= t('delegates_page.acknowledges') %></p>
 
-  <% delegate_sort_order = { region: :asc, delegate_status: :desc, name: :asc } %>
-
-  <%= render partial: "delegates_in_region_table", locals: { delegates: @board_members.order(delegate_sort_order) } %>
+  <% delegate_sort_order = { region: :asc, name: :asc } %>
```

Comment:
Showing delegates before candidates doesn't make much sense to me, since the alphabetical order is broken.
Does anyone know why it is like that?

Reply:
That's just what the original page had when I ported it to Rails. Feel free to change it!

Reply:
Ok, I'll change it to show in alphabetical order inside country/region,